crates/ibc/src/events.rs

```diff
@@ -188,6 +203,48 @@ impl FromStr for IbcEventType {
     }
 }
 
+/// Message event from an IbcEventType
+impl TryFrom<IbcEventType> for abci::Event {
+    type Error = Error;
+
+    fn try_from(event_type: IbcEventType) -> Result<Self, Self::Error> {
```

Comment:
I find this unintuitive since it's not immediately clear to me that the map `IbcEventType -> abci::Event` yields a `message` abci `Event`. Can we instead make this a function 

```rust
fn try_message_event_from_event_type(event_type: IbcEventType) -> Result<abci::Event, Error> { ... }`
```